OneCommunityGlobal/HighestGoodNetworkApp
# TuanDinh_TeamsReportLogs_UnitTest

Add Unit test for TeamsReportLog component

# Description
Create the unit test for TeamsReportLogs component

## Main changes explained:
- Add new file TeamsReportLogs.test.js
- In TeamsReportLogs.test.js, i added 7 test case to test the TeamsReportLogs.jsx component.
- In the begin, test if the component renders without crashing, next test if the component displays the correct title, displays the correct number of members, displays the correct total team blue squares, displays the correct total Weekly Committed Hours, and displays the correct total Teams Work Hours. Finally, test if the component still display empty blocks when no team data is provided
…

## How to test:
1. check into current branch
2. do `npm install`
3. do `npm test TeamsReportLogs.test.js` to run the test
4. Verify that all 7 tests have passed

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/81892917/db28eb5e-d25e-4266-b1f8-32ba05edcf7e



## Note:
Include the information the reviewers need to know.
